pkg/controller/scheduledjob/utils.go

```diff
@@ -172,10 +172,6 @@ func getRecentUnmetScheduleTimes(sj batch.ScheduledJob, now time.Time) ([]time.T
 	return starts, nil
 }
 
-func isJobActive(j *batch.Job) bool {
-	return !isJobFinished(j)
-}
-
 func isJobFinished(j *batch.Job) bool {
```

Comment:
Since this method is copy&paste from [this one](https://github.com/kubernetes/kubernetes/blob/42000793a6f3882199bb581d938ff3a118af8837/pkg/controller/job/jobcontroller.go#L562). Can we export the other one and use it instead?